pagopa/io-app
# feat: [IOBP-421] Added additional outcome screen for already added methods

## ⚠️ This PR depends on #5243 and https://github.com/pagopa/io-dev-api-server/pull/324 ⚠️

## Short description
This PR adds a new outcome screen for an already onboarded payment method

## List of changes proposed in this pull request
- Added translation locales for the screen;
- Added `ALREADY_ONBOARDED` outcome into enum;
- Added `ALREADY_ONBOARDED` pictogram into definition; 

## How to test
- Checkout https://github.com/pagopa/io-dev-api-server/pull/324 and run dev-server
- Start the new onboarding process from dev mode playground;
- Select from the dropdown the outcome `ALREADY_ONBOARDED`;
- You should navigate to the correct page showed in the preview below

## Preview
<img src="https://github.com/pagopa/io-app/assets/34343582/ef560287-7bd3-4f45-a955-0c248f41d9b3" width="250" />
